specs-go/config.go

```diff
@@ -126,7 +126,7 @@ type Hook struct {
 	Path    string   `json:"path"`
 	Args    []string `json:"args,omitempty"`
 	Env     []string `json:"env,omitempty"`
-	Timeout *int     `json:"timeout,omitempty"`
```

Comment:
Add back the pointer to this field so we can validate a actual user provided 0 value.